### DISCUSSION TITLE
Installation of other software packages inside this image

### DISCUSSION THREAD

**Discussion by techworldthink** [Ysurac/openmptcprouter#1776] [Q&A]
is it possible to install other software packages inside this image? (i use Raspberry.)

**Comment by Ysurac** [comment]
OpenMPTCProuter is based on OpenWrt, so you can install all OpenWrt availables packages via System->Software menu.

**Reply by techworldthink** [reply]
Thankyou

**Reply by Bumpalert** [reply]
@Ysurac i was trying to install tcpdump, netstat -m, "https://github.com/nasa/multipath-tcp-tools/ " etc to debug as suggested in  http://multipath-tcp.org/pmwiki.php/Users/Tools. Can I scp to the ssh and install from the external git repositories? Or any equivalents already available in omr package? I am still debugging a mptcp not working issue,which I reported earlier on https://github.com/Ysurac/openmptcprouter/discussions/1758

**Reply by Ysurac** [reply]
You can't compile inside OpenMPTCProuter.

**Reply by techworldthink** [reply]
is there any way to change matric value in routing table?

**Reply by Ysurac** [reply]
It's not a good idea if you want OpenMPTCProuter to work correctly.

**Reply by techworldthink** [reply]
I have eth1 and eth2 interfaces. eth1 has matric value of 3 and eth2 has 4. when i run omr-speedtest more traffic go through eth1.so,
what should i need to do ? (for equally prioritize these two interfaces)

**Reply by Ysurac** [reply]
You can't. Priority go to lower latency interface.

**Reply by techworldthink** [reply]
ok thankyou

**Reply by techworldthink** [reply]
How to find the MPTCP version that i have installed by OpenMPTCPRouter?

**Reply by Ysurac** [reply]
It's MPTCP from master branch with some patches.

**Reply by techworldthink** [reply]
Is there any guide available to configure OpenMPTCPRouter ? There is a lot of settings and configuration is avaible, I am little confused..

**Reply by Ysurac** [reply]
Keep default settings ;)

**Reply by techworldthink** [reply]
can you please tell me the difference of master,backup,enable ?

**Reply by Ysurac** [reply]
Master is the connection used for initial proxy/vpn connection. Should be a stable connection.
Enable is a connection that can be aggregated.
Backup is a connection that should be used only when Master/Enabled connections are down.

**Reply by techworldthink** [reply]
Thanks a lot.

**Reply by techworldthink** [reply]
I found tun0 has 2 entries in route table with different matric values (1200 , 0).  why this happen?

**Reply by Ysurac** [reply]
It's not a problem. I don't remove default entry when I set the VPN as default route with metric 0.

**Reply by techworldthink** [reply]
ifconfig and route commands give different matric values for all interfaces. Incase of ifconfig , matric values of all interfaces is 1. But route commands give different values. Can you tell me why?

**Reply by Ysurac** [reply]
route table number is used as metric for each interface.
